lib/graphql/execution/execute.rb

```diff
@@ -76,10 +76,11 @@ def resolve_field(owner, selection, parent_type, field, object, query_ctx)
 
         lazy_method_name = query.lazy_method(raw_value)
         result = if lazy_method_name
```

Comment:
Hmm, `lazy_method_name` is a throw-away, maybe I can find a way to search for it only once.